lib/osf-components/addon/components/license-picker/component.ts

```diff
@@ -22,11 +23,13 @@ export default class LicensePicker extends Component {
     @service analytics!: Analytics;
     @service store!: DS.Store;
     @service theme!: Theme;
+    @service intl!: Intl;
 
     showText: boolean = false;
     node: Node = this.node;
     licensesAcceptable!: QueryHasManyResult<License>;
     helpLink: string = 'https://openscience.zendesk.com/hc/en-us/articles/360019739014';
+    placeholder: string = this.intl.t('registries.registration_metadata.select_license');
```

Comment:
Now that I'm looking at this closer, since this license picker should be a generic component, the translation key shouldn't be buried under `registries.registration_metadata`. That being said, the translations need a lot of re-organization anyways...